src/Symfony/Component/HttpFoundation/ParameterBag.php

```diff
@@ -188,6 +188,39 @@ public function getBoolean($key, $default = false)
     }
 
     /**
+     * Returns the parameter value converted to a DateTime object.
+     *
+     * @param string                    $key      The parameter key
+     * @param string                    $format   The expected date format
+     * @param \DateTimeInterface|string $default  The default value to be converted to a DateTime object if the parameter key does not exist
+     * @param \DateTimeZone|null        $timeZone A DateTimeZone object representing the desired time zone
+     *
+     * @return \DateTimeInterface|null
+     */
+    public function getDate($key, $format, $default = null, \DateTimeZone $timeZone = null)
+    {
+        $time = $this->get($key, $default);
+
+        if ($time instanceof \DateTimeInterface) {
```

Comment:
That might be added as an early return condition, currently the function works as expected tho, as `null` will be passed to the `createFromFormat` which will not return a valid date.